packages/azure_frontdoor/data_stream/access/_dev/test/pipeline/test-fdaccess.log-expected.json

```
@@ -0,0 +1,271 @@
+{
+    "expected": [
+        {
+            "@timestamp": "2021-12-15T03:10:11.647Z",
+            "azure": {
+                "frontdoor": {
+                    "access": {
+                        "backend_hostname": "vantagev6erp.azurewebsites.net:443",
+                        "cache_status": "CONFIG_NOCACHE",
+                        "error_info": "NoError",
+                        "is_received_from_client": true,
+                        "pop": "SIN",
+                        "routing_rule_name": "erp",
+                        "rules_engine_match_names": [],
+                        "time_taken": "0.384",
+                        "time_to_first_byte": "0.384"
+                    },
+                    "category": "FrontdoorAccessLog",
+                    "operation_name": "Microsoft.Network/FrontDoor/AccessLog/Write",
+                    "resource_id": "/SUBSCRIPTIONS/49D1B571-1CBE-402D-B523-AFEE3C19B64E/RESOURCEGROUPS/WAF-RG/PROVIDERS/MICROSOFT.NETWORK/FRONTDOORS/MITCLOUD",
+                    "tracking_reference": "0k1y5YQAAAAAWd0Uc6UcnR7WN8uo2prYZU0lOMzBFREdFMDIxNwBkYjIxMTMyNi1mZTJmLTQwYWYtOTA4My03MGUyYTJmYWRmZjc="
+                }
+            },
+            "client": {
+                "address": "89.160.20.128",
+                "ip": "89.160.20.128",
+                "port": 50382
+            },
+            "cloud": {
+                "provider": "azure"
+            },
+            "ecs": {
+                "version": "1.12.0"
```

```suggestion
                "version": "8.5.0"
```
The "*-expected.json" files need to be all updated with the correct ecs version